src/App.js

```diff
@@ -21,7 +26,12 @@ export default function App() {
         <Route path="/sometest" element={<Testing />} />
         <Route path="/rsvp-preview" element={<RSVP />} />
         <Route path="/invite" element={<InvitationForm />} />
+<<<<<<< HEAD
```

Comment:
Unsure why Head comment is here

Reply:
Yeah im not sure why but it seems like the code in my local repo is different from the one I pushed up. I was able to rebase just fine and see the registry route and folder on my local machine with all of my code. Also that HEAD line is not showing up on mine at all. Ill try and sort this out